pkg/sysinfo/sysinfo.go

```diff
@@ -45,5 +43,15 @@ func New(quiet bool) *SysInfo {
 	} else {
 		sysInfo.AppArmor = true
 	}
+
+	// Check if Devices cgroup is mounted, it is hard requirement for container security.
+	if _, err := cgroups.FindCgroupMountpoint("devices"); err != nil {
+		if cgroups.IsNotFound(err) {
```

Comment:
perhaps since these are both Fatals, just do `log.Fatalf("error mounting devices cgroup: %v", err)` and remove the else.